### PR TITLE
Feat/chore/update design system to v1

### PR DESCRIPTION
**Pull request by sonwit**
## Description
Update DS to v1 beta 

## Issue
#1071 

https://docs.google.com/document/d/1jBOt0YjM9sJy4fyf0Y2mqEvwalXAU7Tp2B72Ip9f89k/edit?usp=sharing

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/components/Filter/OptionDisplay.module.css [hunk 1]
```diff
@@ -9,16 +9,16 @@
...
 .option {
   width: 100%;
   border: none;
-  border-bottom: 1px solid var(--fds-semantic-border-divider-default);
-  background-color: var(--fds-semantic-background-default);
+  border-bottom: 1px solid var(--ds-color-neutral-contrast-subtle);
+  background-color: var(--ds-color-neutral-contrast-default);
   cursor: pointer;
   font: inherit;
   text-align: left;
```
Comment (allinox) [inline src/components/Filter/OptionDisplay.module.css:24]: For the line under:
The options of the filter look a bit smaller than before for some reason.
I suggest changing to `padding: 8px 0;`

File: src/components/Filter/OptionDisplay.module.css [hunk 2]
```diff
@@ -32,26 +32,25 @@
...
-  box-shadow: inset 0 0 0 2px var(--fds-semantic-border-input-hover);
+  box-shadow: inset 0 0 0 2px var(--ds-color-accent-border-default);
 }
 
 .option:hover {
-  background-color: var(--fds-semantic-surface-info-subtle);
-  border-bottom: 2px solid var(--fds-semantic-border-input-hover);
+  background-color: var(--ds-global-blue-2);
+  border-bottom: 2px solid var(--ds-color-accent-border-default);
   padding: 4px 0 3px 0;
```
Comment (allinox) [inline src/components/Filter/OptionDisplay.module.css:41]: For the same reason as above
```suggestion
  padding: 8px 0 7px 0;
```

File: src/features/singleRight/delegate/ChooseRightsPage/ChooseRightsPage.tsx [hunk 3]
```diff
@@ -225,12 +225,11 @@ export const ChooseRightsPage = () => {
...
               </Button>
             </div>
-          </Popover.Content>
-        </Popover>
+          </Popover>
+        </Popover.Context>
         <Button
           variant='secondary'
-          color='first'
-          fullWidth={isSm}
```
Comment (allinox) [inline src/features/singleRight/delegate/ChooseRightsPage/ChooseRightsPage.tsx:233]: I'm guessing `fullWidth` is not a thing anymore?
Should we then add some manual styling to make the buttons take up all horizontal space on small screens?
image

File: src/features/singleRight/delegate/ReceiptPage/ReceiptPage.module.css
Comment (allinox) [inline src/features/singleRight/delegate/ReceiptPage/ReceiptPage.module.css:1]: Notice some different coloring here.
Don't know why though.. 😅
image

Comment (allinox) [reply src/features/singleRight/delegate/ReceiptPage/ReceiptPage.module.css:1]: image

**Review by allinox — APPROVED** [review]
This was a huge task, and you completed it brilliantly! ✨

Amazing work on adding storybook - complete with a solution for mocking calls!
Almost everything looks like before in the solution as well, which is a testament to how darn thorough you've been when making the transition to the new design system version. Thank you for that!

I have just a couple of comments, of which none are critical, so I'm just gonna approve right away so you can merge whenever you've gotten to them.
Thanks again for the stellar work! 🌟

File: src/features/amUI/common/List/List.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  spacing: true,
+  background: true,
+};
+
+export default {
+  title: 'Features/AMUI/List',
+  component: List,
+  argTypes: {
+    borderStyle: {
+      options: ['solid', 'dashed'],
```
[the remark this answers is not in the document]
Comment (sonwit) [reply src/features/amUI/common/List/List.stories.tsx:27]: Absolutely right. I'll remove it. Probably just copy-paste from the other list.

File: src/components/Filter/OptionDisplay.module.css (hunk 1 above)
Comment (sonwit) [reply src/components/Filter/OptionDisplay.module.css:24]: 👌

File: src/features/singleRight/delegate/ReceiptPage/ReceiptPage.module.css
Comment (sonwit) [reply src/features/singleRight/delegate/ReceiptPage/ReceiptPage.module.css:1]: Great catch! I removed the extra error-summary wrapping the text/icon. Looks a little better now :)
Skjermbilde 2024-10-08 kl  15 58 49

**Pull request merged**